src/quo/components/inputs/amount_input/style.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo.components.inputs.amount-input.style
+  (:require
+    [quo.foundations.colors :as colors]
+    [react-native.platform :as platform]))
+
+(defn get-selection-color
+  [customization-color theme]
+  (colors/alpha
+   (colors/resolve-color customization-color
+                         theme
+                         (if (= :dark theme) 60 50))
+   (if platform/ios? 1 0.2)))
+
+(def container
+  {:flex-direction  :row
+   :justify-content :center
+   :align-items     :center})
+
+(def input-container {:flex 1})
+
+(defn input-text
+  [theme type]
+  {:padding 0
+   :color   (if (= type :error)
+              colors/danger-60
```

Comment:
please use `(colors/resolve-color :danger theme 60)` in place of colors/danger-60

Reply:
Done ✅

Reply:
BTW, @J-Son89, looks like the third parameter of the function is for the opacity, and according to the Figma the opacity should be 100%, so I'll remove param "60". The function will look next `(colors/resolve-color :danger theme)`.

Reply:
Sorry yes you are correct, my bad for the confusion 😅🙏

Reply:
No worries 😃